apple/internal/resource_actions/plist.bzl

```
@@ -275,7 +275,8 @@ def merge_root_infoplists(
     # Info.plists out of the box coming from Xcode.
     substitutions["DEVELOPMENT_LANGUAGE"] = "en"
 
-    if include_executable_name and executable_name:
+    executable_name = executable_name or bundle_name
+    if include_executable_name:
```

This is a behavior change, right?

If it's desired, can we do this?
```suggestion
    if include_executable_name:
        executable_name = executable_name or bundle_name
```
If it's not, how about this?
```suggestion
    executable_name = executable_name or bundle_name
    if include_executable_name and executable_name:
```

---

i don't think so because previously every declared executable_name always fell back to bundle_name, and i don't think it's valid to not have a bundle_name?